.github/ISSUE_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@ Feature Requests:
   Most features should start as plugins outside of jQuery.
 
 Bug Reports:
-  Note that we only can fix bugs in the latest (1.x, 2.x, 3.x) versions of jQuery.
+  Note that we only can fix bugs in the latest version of jQuery.
```

Comment:
Although see what I've written in the commit message:
> We no longer support jQuery 1.x/2.x and mentioning 3.x would just mean the text
gets out of date once we release jQuery 4.

So I did this on purpose so that we don't have to update this text once 4.0 comes out. Do you still prefer to change it?